system-test/storage.ts

```diff
@@ -2516,33 +2516,6 @@ describe('storage', () => {
       await file.delete();
     });
 
-    it('should skip validation if file is served decompressed', async () => {
-      const filename = 'logo-gzipped.png';
-      await bucket.upload(FILES.logo.path, {destination: filename, gzip: true});
-
-      tmp.setGracefulCleanup();
-      const {name: tmpFilePath} = tmp.fileSync();
-
-      const file = bucket.file(filename);
-
-      await new Promise((resolve, reject) => {
-        file
-          .createReadStream()
-          .on('error', reject)
-          .on('response', raw => {
-            assert.strictEqual(
-              raw.toJSON().headers['content-encoding'],
-              undefined
-            );
-          })
-          .pipe(fs.createWriteStream(tmpFilePath))
-          .on('error', reject)
-          .on('finish', resolve);
-      });
-
-      await file.delete();
-    });
```

Comment:
I think the opposite of this test should exist

Reply:
this test is actually still needed. i had misunderstood the implementation